### PR TITLE
updated start work dialog -> to ask hunter for action plan

### PR DESCRIPTION
**Pull request by thelostone-mc**
##### Description
- removes do you have a question 
- renames issue description label -> action plan
- updated alert to use our custom alerts
- action plan is a compulsory field 

##### Checklist

- [x] linter status: 100% pass
- [x] changes don't break existing behavior
- [x] commit message follows [commit guidelines](https://github.com/gitcoinco/web/blob/master/docs/CONTRIBUTING.md#step-4-commit)

##### Affected core subsystem(s)
- html
- js
- postgres

##### Testing
x

@mbeacom didn't test the github publish feature

@vs77bb thanks for pointing this out, managed to fix a few more things along the way  :P

### CODE REVIEW

File: app/assets/v2/js/pages/bounty_details.js [hunk 1]
```diff
@@ -429,25 +429,27 @@ var show_interest_modal = function() {
...
-        var has_question = event.target[0].value;
-        var issue_message = event.target[2].value;
+        var issue_message = event.target[0].value;
+        var agree_precedence = event.target[1].checked;
+        var agree_not_to_abandon = event.target[2].checked;
 
-        var agree_precedence = event.target[3].checked;
-        var agree_not_to_abandon = event.target[4].checked;
+        if (!issue_message) {
+          _alert({message: 'Please provide an action plan for this ticket.'}, 'error');
```
Comment (mbeacom) [inline app/assets/v2/js/pages/bounty_details.js]: Should we `gettext()` these message strings?

Comment (thelostone-mc) [reply app/assets/v2/js/pages/bounty_details.js]: we should :P

Comment (mbeacom) [reply app/assets/v2/js/pages/bounty_details.js]: 😭

**Comment by thelostone-mc** [comment]
@mbeacom done! PS: how do i test the github publish feature  ?

File: app/assets/v2/js/pages/bounty_details.js (hunk 1 above)
Comment (thelostone-mc) [reply app/assets/v2/js/pages/bounty_details.js]: :trollface: 
eyeofsauron

**Comment by mbeacom** [comment]
@vs77bb I'll check if we're doing that for the start work comment and raise an issue to add it if not.  Good call!

**Comment by mbeacom** [comment]
@vs77bb 
screenshot 2018-05-09 16 27 47

`Test` was my input on this dialog.

**Pull request merged**